src/Backups/BackupsWorker.h

```diff
@@ -90,6 +90,7 @@ class BackupsWorker
 
     std::unique_ptr<ThreadPool> backups_thread_pool;
     std::unique_ptr<ThreadPool> restores_thread_pool;
+    std::unique_ptr<ThreadPool> entries_collector_thread_pool;
```

Comment:
Collecting of backup entries is not performed in parallel with writing them to a backup. So you don't need another thread pool here, just use `backups_thread_pool`.